src/utils/compoundApyHelpers.ts

```diff
@@ -0,0 +1,15 @@
+export const calculateCakePerThousand = (numberOfDays, apy, cakePrice) => {
+  const timesCompounded = 365
+  const apyAsDecimal = apy / 100
+  const daysAsDecimalOfYear = numberOfDays / 365
```

Comment:
I guess this 365 is the same as timesCompounded ?

Reply:
PS: Can you add a comment header on each of these function, and add unit tests

Reply:
Done. Writing tests for the functions improved the functions a lot, too. 👍